stt_engines/deepspeech/main.sh

```diff
@@ -0,0 +1,44 @@
+#!/bin/bash
+
+stt_deepspeech_install () {
+  set -e # Don't attempt to continue if an error occurs
+  jv_install python3 python3-pip virtualenv
+  cd "$(dirname "${BASH_SOURCE[0]}")"
+  virtualenv -p python3 .
+  source bin/activate
+  pip3 install deepspeech
+  deactivate
+  wget https://github.com/mozilla/DeepSpeech/releases/download/v0.7.0/deepspeech-0.7.0-models.pbmm
+  wget https://github.com/mozilla/DeepSpeech/releases/download/v0.7.0/deepspeech-0.7.0-models.scorer
+  cd "$jv_dir"
+}
```

Comment:
- Make sure that Python3, pip3 and virtualenv are installed
- cd into "stt_engines/deepspeech"
- create a virtualenv (that uses Python3)
- activate said virtualenv
- install deepspeech via  pip
- deactivate the virtualenv
- download the model and scorer from Mozilla's GitHub repository
- cd into $jv_dir